db/migrate/20170607101716_create_themes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateThemes < ActiveRecord::Migration[5.1]
+  def change
```

Comment:
Hi @tuzz, sorry I didn't talk to you about `Groups` before.

In the Audit Tool, we started thinking about Groups as a way to build a static list (Themes/Subthemes) of Content Items to be Audited by Content Designers. After a few conversations with Natalie, it felt that the CPM would also benefit from having groups of Content Items: Content Designers could create them to measure their performance over time.

Groups exist in the CPM as an API endpoint because the first idea we explored was to expose an API for other teams so they can build the themes via an external tool, but this plan was discarded after a few weeks.

In the Audit Tool, we are now adding filtering rules to Themes and Subthemes, and it feels to me that Themes and Subthemes are quite related to Groups/Subgroups, as I can't see any specific behaviour that justifies having separate Tables in the database. We could model them as domain objects, but it could be worth exploring reusing the Group/Subgroups infrastructure that is already in place.

My understanding is that Theme/Subthemes are a subtype of a Group. In the same way that the Term Generation team has spiked modelling with a Group Sample of Content Items to apply term generation. 

The current implementation of Groups will allow us to have a hierarchical structure (of many levels if needed) in case we want to split Sub-themes in the future, and also assigning individual Content Items to a Group which is also aligned with some user research from Natalie/Joe

Filtering would need to be done once (in the Query), although the `Search` API could perfectly speak of Themes and Subthemes.

There is a lot of work already done, so we could address this refactor in a future PR.